tap-snapshots/test/check/snapshots.js.test.cjs

```diff
@@ -250,7 +250,7 @@ The following required devDependencies were not found:
   @npmcli/eslint-config
   tap
 
-To correct it: npm rm @npmcli/template-oss @npmcli/eslint-config tap && npm i @npmcli/eslint-config@latest tap@latest --save-dev && npm i @npmcli/template-oss@{{VERSION}} --save-dev --save-exact
+To correct it: npm rm @npmcli/template-oss @npmcli/eslint-config tap && npm i @npmcli/eslint-config@* tap@* --save-dev && npm i @npmcli/template-oss@{{VERSION}} --save-dev --save-exact
```

Comment:
I think the output from these messages should still say `@latest`.

Reply:
Why?